src/Mailgun/Resource/Api/Message/ShowResponse.php

```diff
@@ -371,14 +363,6 @@ public function getContentIdMap()
     }
 
     /**
-     * @param string $contentIdMap
-     */
-    private function setContentIdMap($contentIdMap)
```

Comment:
Not sure this should be removed, it should be used instead... =)

Same with setAttachment.